graphistry/validate/validate_graph.py

```diff
@@ -0,0 +1,61 @@
+def check_node_dataframe_exists(g, verbose=True):
+    if g._nodes is None:
+        if verbose:
+            print("Warning: graph was created with only edges. Skipping Node ID check if Node IDs match edge IDs. Use g2 = g.materialize_nodes() to force node df creation. Exiting.")
```

Comment:
switch from `print` statements to `warn` + `logger.info` depending on use

removes needed for `verbose` param